src/export/digitalPlanning/model.ts

```diff
@@ -254,5 +429,273 @@ export class DigitalPlanning {
         ),
       },
     };
+
+    if (this.passport.data?.["application.fee.reference.govPay"]) {
+      set(
+        baseFee,
+        "reference.govPay",
+        this.passport.data?.["application.fee.reference.govPay"]?.[
+          "payment_id"
+        ],
+      );
+    }
+
+    return baseFee;
+  }
+
+  private getApplicationDeclaration(): Payload["data"]["application"]["declaration"] {
+    return {
+      accurate: Boolean(
```

Comment:
Thanks for catching this! Now updated. Because this schema is a bit stricter than OneApp, I've repeated a private function definition that can return a strict `boolean` rather than `boolean | undefined` as OneApp does, while still properly catching/distinguishing between "true" & "false" :+1: